smarts/core/controllers/trajectory_tracking_controller.py

```diff
@@ -208,14 +208,33 @@ def perform_trajectory_tracking_PD(
         lateral_gain = lerp(3, final_lateral_gain, normalized_speed)
         heading_gain = lerp(0.03, final_heading_gain, normalized_speed)
         steering_filter_constant = lerp(
-            2, final_steering_filter_constant, normalized_speed
+            12, final_steering_filter_constant, normalized_speed
         )
-        heading_error_derivative_gain = lerp(
+        heading_error_derivative_gain = 0 * lerp(
             1.5, final_heading_error_derivative_gain, normalized_speed
         )
-        lateral_error_derivative_gain = lerp(
+        lateral_error_derivative_gain = 0 * lerp(
```

Comment:
is `lateral_error_derivative_gain` and `heading_error_derivative_gain` at line 213 and 216 no longer needed since it is set at 223 and 224?

Reply:
Nice catch.